cmd/demo/node.go

```diff
@@ -448,9 +451,9 @@ func (n *node) Info(args []string) error {
 		if peer.ch == nil {
 			fmt.Fprintf(w, "%s\t%s\t\n", alias, "Connected")
 		} else {
-			my, other := peer.ch.GetBalances()
+			bals := weiToEther(peer.ch.GetBalances())
 			fmt.Fprintf(w, "%s\t%v\t%d\t%v\t%v\t\n",
-				alias, peer.ch.Phase(), peer.ch.State().Version, my, other)
+				alias, peer.ch.Phase(), peer.ch.State().Version, bals[0], bals[1])
```

Comment:
```suggestion
				alias, ch.Phase(), ch.State().Version, bals[ch.Idx()], bals[1-ch.Idx()])
```
And at the beginning of the loop, define `ch := peer.ch`.

I think this is a bug, @ggwpez ?